unittests/testaccess_policies.py

```diff
@@ -125,7 +125,7 @@ def test_access_policies_create(self, mock_post):
             msg="CREATE method called incorrectly",
         )
 
-    @mock.patch("requests.get")
+    @mock.patch("requests.Session.get")
```

Comment:
probabky better ro mock self.arch._session usin the with syntax instead of a decorator

       with mock.patch(self.arch._session.get) as mock_get:

Reply:
try it for one tests - if no problems change the rest...

this is **much** better as you are mocking where it is used instead of where it is defiled

Reply:
Not 100% sure if this is the correct usage of the syntax
```python


    def test_access_policies_read(self):
        """
        Test access_policy reading
        """
        with mock.patch(self.arch._session.get) as mock_get:
            mock_get.return_value = MockResponse(200, **RESPONSE)

            access_policy = self.arch.access_policies.read(IDENTITY)
            self.assertEqual(
                tuple(mock_get.call_args),
                (
                    ((f"url/{ROOT}/{ACCESS_POLICIES_SUBPATH}/{IDENTITY}"),),
                    {
                        "headers": {
                            "content-type": "application/json",
                            "authorization": "Bearer authauthauth",
                        },
                        "verify": True,
                        "cert": None,
                    },
                ),
                msg="GET method called incorrectly",
            )
```

but I get the following error
``` 
ERROR: test_access_policies_read (unittests.testaccess_policies.TestAccessPolicies)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/builder/unittests/testaccess_policies.py", line 132, in test_access_policies_read
    with mock.patch(self.arch._session.get) as mock_get:
  File "/usr/local/lib/python3.6/unittest/mock.py", line 1548, in patch
    getter, attribute = _get_target(target)
  File "/usr/local/lib/python3.6/unittest/mock.py", line 1397, in _get_target
    target, attribute = target.rsplit('.', 1)
AttributeError: 'function' object has no attribute 'rsplit'
```